controller/example.controller.ts

```diff
@@ -0,0 +1,18 @@
+/*
+
+    THIS IS AN EXAMPLE FOR CONTROLLER. PLEASE DONT DELETE THIS FILE FOR 2 WEEKS
+
+*/    
+
+
+const books:any[] = [];
```

Comment:
space books: any[]
what is books?

Reply:
don't use any

Reply:
This is just an example to show how they should create endpoints